app/controllers/api/v1/jobs.rb

```diff
@@ -0,0 +1,30 @@
+module API
+  module V1
+    class Jobs < Base
+
+        resources :jobs do
+          desc 'return all of the jobs'
+          get do
+            Job.all
+          end
+
+        desc "Return a job"
+        params do
+          requires :id, type: String, desc: "ID of the
+          job"
+        end
+        get '/:id' do
+          Job.find(params[:id])
```

Comment:
Please add some responses, now you are going to get 500 if you pass an incorrect id. I can't find specs here :(